rules/windows/credential_access_generic_localdumps.toml

```diff
@@ -0,0 +1,56 @@
+[metadata]
+creation_date = "2022/08/28"
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
```

Comment:
@Mikaayenson @terrancedejesus do I need to min_stack new rules?

Reply:
Yes, min stack 8.3.0.

Reply:
Those would still make it to the other versions? Or should I push without the min_stack and then add it?

Reply:
It should fail a unit test if you push without a min stack because at build time, the build fields will be added and it will check for a min_stack_version tag.

This rule will only be available on stacks 8.3 and up.